cpu/podman_container.py

```diff
@@ -0,0 +1,137 @@
+                self.cancel('%s is needed for the test to be run' % package)
+
+        url = ("http://sourceforge.net/projects/ebizzy/files/ebizzy"
+               "/0.3/ebizzy-0.3.tar.gz")
+
+        tarball = self.fetch_asset(self.params.get("ebizy_url", default=url))
+        archive.extract(tarball, self.workdir)
+        version = os.path.basename(tarball.split('.tar.')[0])
+
+        self.sourcedir = os.path.join(self.workdir, version)
+        os.chdir(self.sourcedir)
+        process.run('[ -x configure ] && ./configure', shell=True)
+        build.make(self.sourcedir)
+        self.cpu_quota = self.params.get("cpu_quota", default=50000)
+        self.cpu_period = self.params.get("cpu_period", default=100000)
+        self.workload_run_time = self.params.get("run_time", default=1000)
+
+    def Generate_docker_file(self):
+        """
+        Generating the docker image with ebizzy binary attached
+        """
+
+        self.log.info("Inside the Generate_docker_file---")
+        Docker_row_data = "FROM busybox\nCOPY ebizzy \
+                /usr/local/bin/ebizzy\nRUN chmod +x /usr/local/bin/ebizzy"
+        with open("Dockerfile", 'w') as file_obj:
+            file_obj.write(Docker_row_data)
+
+    def Build_docker_image(self):
+        """
+        Building the docker image with ebizzy workload so that the
+        podman container will fetch the ebizzy image at run time
+        """
+        cmd = "podman build -t custom-busybox-ebizzy ."
+        process.system(cmd,
+                       ignore_status=True, shell=False, sudo=True)
+
+    def run_container(self):
+        """
+        Running the podman container and attaching ebizzy workload
+        """
+        workload_path = "/usr/local/bin/ebizzy"
+        cmd = "podman run --detach --cpu-period=%s --cpu-quota=%s \
+                custom-busybox-ebizzy %s -S %s" % (self.cpu_period,
+                                                   self.cpu_quota,
+                                                   workload_path,
+                                                   self.workload_run_time)
+
+        process.system(cmd,
+                       ignore_status=True, shell=False, sudo=True)
+
+    def log_generater(self, smt_state):
+        """
+        Capture the podman stats
+        :param smt_state: passing the smt state value to capture the logs
+        for same state.
+        """
+        podman_dir = self.logdir + "/podman"
+        if not os.path.isdir(podman_dir):
+            os.mkdir(podman_dir)
+
+        log_dir = podman_dir + "/" + str(smt_state) + ".txt"
+        cmd = "nohup podman stats &> %s &" % (log_dir)
+        process.run(cmd, shell=True)
+        self.log.info("Waiting for some time dump the podman stats to file")
+        time.sleep(30)
+        self.log.info("Killing the podman stats command PID")
+        cmd = 'pgrep -f "podman stats"'
+        cmd_output = process.run(cmd, shell=True)
+        process_pid = cmd_output.stdout
```

Comment:
add this .stdout `  cmd_output = process.run(cmd, shell=True)`
like `  cmd_output = process.run(cmd, shell=True).stdout`